identity-apps-core/components/org.wso2.identity.apps.common/src/main/java/org/wso2/identity/apps/common/listner/AppPortalRoleManagementListener.java

```diff
@@ -100,6 +102,15 @@ public void preUpdateUserListOfRole(String roleId, List<String> newUserIDList, L
             return;
         }
 
+        try {
+            if (isRoleModificationAllowedForTenant(tenantDomain)) {
```

Comment:
```suggestion
            if (OrganizationManagementUtil.isOrganization(tenantDomain)) {
```

Additional private method is not required for this case

Reply:
The code is refactored. Removed the unnecessary private method.